src/shared/features/AppletSettings/DeleteAppletSetting/DeleteAppletSetting.test.tsx

```diff
@@ -0,0 +1,81 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import mockAxios from 'jest-mock-axios';
+
+import { SettingParam, renderWithProviders } from 'shared/utils';
+
+import { initialStateData } from 'redux/modules';
+import { page } from 'resources';
+import { Roles } from 'shared/consts';
+import * as encryptionFunctions from 'shared/utils/encryption';
+import { mockedAppletId, mockedCurrentWorkspace, mockedApplet, mockedPassword } from 'shared/mock';
+
+import { DeleteAppletSetting } from './DeleteAppletSetting';
+
+const route = `/dashboard/${mockedAppletId}/settings/${SettingParam.DeleteApplet}`;
+const routePath = page.appletSettingsItem;
+const preloadedState = {
+  workspaces: {
+    workspaces: initialStateData,
+    currentWorkspace: {
+      ...initialStateData,
+      ...mockedCurrentWorkspace,
+    },
+    roles: {
+      ...initialStateData,
+      data: {
+        [mockedAppletId]: [Roles.Manager],
+      },
+    },
+    applet: mockedApplet,
+    workspacesRoles: initialStateData,
+  },
+  applet: {
+    applet: {
+      ...initialStateData,
+      data: mockedApplet,
+    },
+  },
+  popups: {
+    data: {
+      applet: mockedApplet,
+      encryption: null,
+      popupProps: undefined,
+      deletePopupVisible: true,
+      duplicatePopupsVisible: false,
+      transferOwnershipPopupVisible: false,
+      publishConcealPopupVisible: false,
+    },
+  },
+};
+const getPublicKeyMock = () => Buffer.from(JSON.parse(mockedApplet?.encryption?.publicKey || ''));
+
+const mockedUseNavigate = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedUseNavigate,
+}));
+
+describe('DeleteAppletSetting', () => {
+  test('should render and submit', async () => {
+    mockAxios.delete.mockResolvedValueOnce(null);
+    jest.spyOn(encryptionFunctions, 'getAppletEncryptionInfo').mockImplementation(() => ({
+      getPublicKey: getPublicKeyMock,
+    }));
+    renderWithProviders(<DeleteAppletSetting />, { preloadedState, route, routePath });
+
+    expect(screen.getByTestId('applet-settings-delete-applet-delete-button')).toBeVisible();
+    expect(screen.getByTestId('applet-settings-delete-applet-delete-password-popup')).toBeVisible();
+
```

Comment:
Could you pls add test for incorrect password error?
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/1cf1b27e-90ec-4d34-baf1-70b36913a9c3)

Reply:
such tests should be for DeletePopup component, not DeleteAppletSetting